CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [Dimitris Koutsogiorgas](https://github.com/dnkoutso)
   [#1698](https://github.com/CocoaPods/CocoaPods/issues/1698) 
 
+* Prevents need for .swift-version file in Objective-C pods  
+ +  [Austin Emmons](https://github.com/atreat)
```

Comment:
Whats the reasoning behind this changelog trailing whitespace? I'm curious to know why that is the desired format. (Added in c44c8ff)